package.json

```diff
@@ -12,7 +12,7 @@
     "start:local": "npm run vi && ng serve --configuration=local",
     "start:e2e": "npm run vi && ng serve --configuration=e2e",
     "start:e2e-local": "npm run vi && ng serve --configuration=e2e-local",
-    "build": "npm run vi && ng build --prod",
+    "build": "npm run vi && ng build --prod --tsConfig=src/tsconfig.\"${KUBERMATIC_EDITION:=ee}\".json",
```

Comment:
Thanks to this change we will be able to use `tsconfig.ee.json` or `tsconfig.ce.json` depending on the `KUBERMATIC_EDITION` environment variable. `KUBERMATIC_EDITION` will be defaulted to `ee` if it is not set.